src/Symfony/Component/Serializer/Tests/Normalizer/ObjectNormalizerTest.php

```diff
@@ -953,6 +953,21 @@ public function testDenormalizeWithPropertyPath()
 
         $this->assertEquals($expected, $obj);
     }
+
+    public function testObjectNormalizerWithAttributeLoaderAndObjectHasStaticProperty()
+    {
+        $class = new class {
+            public static string $foo;
+        };
+
+        try {
+            $normalizer = new ObjectNormalizer(new ClassMetadataFactory(new AttributeLoader()));
+            $normalizer->normalize($class);
+            $this->expectNotToPerformAssertions();
+        } catch (\Exception $e) {
+            $this->fail($e->getMessage());
+        }
```

Comment:
You can remove the try-catch, the test will fail no matter what if an exception is raised somewhere